### DISCUSSION TITLE
Plain text output

### DISCUSSION THREAD

**Discussion by subpublic** [remarkjs/remark#1033] [Q&A] [answered]
Hi!

How do I render a plain text version of the markdown content?
Stripping all **, _, [\](\), etc.

**Comment by remcohaszing** [comment, marked as answer]
You can use [mdast-util-to-string](https://github.com/syntax-tree/mdast-util-to-string).

**Reply by subpublic** [reply]
Seems like it should do what I'm aiming for, but I wonder if there could be a way as with rehype plugins (couldn't find any existing)?
I'm using `react-markdown` for my regular rendering, but would like an easy way (plugin) too also render plain text (maybe keep line breaks).

**Reply by wooorm** [reply]
Perhaps... https://github.com/remarkjs/strip-markdown is kinda like that. It turns everything into text and paragraphs. It strips some things. It’s still “markdown”, it’s just that everything is replaced with its contents

**Comment by wooorm** [comment]
It depends on what you mean by “plain-text”. There are several interpretations of that word. Perhaps if you can provide more information, we can help better. See [How do I ask a good question?](https://stackoverflow.com/help/how-to-ask) for more info.
